### PR TITLE
Add hardcase docs, update gitea docs

### PR DESCRIPTION
**Pull request by m1cr0man**
Also standardised daedalus + icarus docs to line up with other hardware docs

### CODE REVIEW

File: docs/services/git.md [hunk 1]
```diff
@@ -6,19 +6,32 @@ Redbrick uses [Gitea](https://gitea.io/en-US/) a community driven fork of
...
+other data is stored in `/var/lib/gitea`
+- The `SECRET_KEY` and `INTERNAL_TOKEN_URI` are stored in `/var/secrets`.
+They are not automatically created and must be copied when setting up new hosts.
+Permissions on the gitea_token.secret must be 740 and owned by git:gitea.
 
-### LDAP Login
+## Redbrick Special Notes
 
-We use ldap auth to login to gitea, the settings are as follows,
-![Gitea ldap settings](/img/gitea-ldap.png)
```
Comment (butlerx) [inline docs/services/git.md:24]: could we start link to nixos configs in the docs? (This can wait for nixos to be migrated to rb org)

**Review by butlerx — DISMISSED** [review]

File: docs/services/git.md (hunk 1 above)
Comment (m1cr0man) [reply docs/services/git.md:24]: Yeah after migration we can do that.

**Comment by m1cr0man** [comment]
Well shit. I just realized I copy-pasted the docs wrong for daedalus icarus. please don't merge.

**Comment by m1cr0man** [comment]
Fixed

**Review by butlerx — APPROVED** [review]

**Review by theycallmemac — APPROVED** [review]

**Pull request merged**